docs/features/asc1/stateless/walkthrough.md

```diff
@@ -241,7 +241,7 @@ For sake of generality the Contract Account ASA Opt-In example makes use of **te
 3. `VAR_TMPL_OPTIN_EXPIRING_BLOCK`: setting a [last valid block](https://developer.algorand.org/docs/features/transactions/#setting-first-and-last-valid) for the ASA Opt-In transaction means that Contract Account will only approve ASA Opt-In transactions until a certain block. This check prevents malicious adversaries from draining Contract Account ALGOs in Opt-In transaction fees after that block.
 
 !!! tip
-  For [AlgoDea](https://algodea-docs.bloxbean.com/) users is convenient to adopt the prefix `VAR_TMPL_` as parameters naming convention so that they can be easily assigned directly within the IDE, before compiling the TEAL stateless program.
+    For [AlgoDea](https://algodea-docs.bloxbean.com/) users is convenient to adopt the prefix `VAR_TMPL_` as parameters naming convention so that they can be easily assigned directly within the IDE, before compiling the TEAL stateless program.
```

Comment:
Should this be: "For [AlgoDea](https://algodea-docs.bloxbean.com/) users, it is convenient to adopt the prefix `VAR_TMPL_` as a parameter naming convention so that they can be easily assigned directly within the IDE, before compiling the TEAL stateless program."

Seemed to be missing a couple words. Hope adding them has intended meaning.